005-tailwindcss-framework.md

```diff
@@ -0,0 +1,38 @@
+# Tailwind CSS Framework
+
+We should implement the use of [Tailwind CSS Framework](https://tailwindcss.com/) on our user and admin interfaces to help simplify front-end development across our team.
+
+**Discussion:** <https://github.com/DoSomething/rfcs/pull/5>
+
+## Problem
+
+It has become more difficult over time for our tech team to confidently update the styling and design patterns of our front-end user interfaces.
+
+While our pattern library, [Forge](https://github.com/DoSomething/forge), worked great for building interfaces with the established patterns it provided, it has not been updated in some time and with our rapidly changing design needs and team structure, it no longer seems like the ideal solution to continue evolving.
```

Comment:
Yeah, I think one of the biggest pain points with Forge was the "library" dev workflow – having to make changes in Forge (and figure out how to `npm link` to see them in Phoenix or wherever) and then publish a release before including the changes in the app. (And something we've had similar pain with elsewhere, such as [this fix for a Gateway issue in Phoenix](https://github.com/DoSomething/phoenix-next/blob/5a63326ea1158022c8096b63c8520198eca803f8/app/Repositories/PostRepository.php#L46-L61)).

(And that's not to say that making it a library wasn't worthwhile – I think it forced us to take more of a "systems" approach to building interfaces then than we have since. And while we weren't good at extracting patterns later on, it was certainly nice to have that shared "baseline" of patterns in every app we work on!)